### DISCUSSION TITLE
does it track domain in activity from member?

### DISCUSSION THREAD

**Discussion by Owned67** [Commandserver/Antispambot#15] [Ideas]
Hello,

does it track domain in activity from member?

**Comment by Commandserver** [comment]
currently not. But it's a good idea actually

**Reply by Owned67** [reply]
and if you can add this for About me user maybe?

will be great <3

Thanks very much

**Reply by Commandserver** [reply]
I'll think about it

**Comment by Commandserver** [comment]
I'm sorry, i can't make that yet. Sounds interesting but i think it's an edge case and not needed for most people

**Reply by Owned67** [reply]
ok no worries.
